### PR TITLE
Minor code fixes

### PR DESCRIPTION
**Pull request by steph-en-m**
Some f-strings in `syft/exceptions.py` weren't properly implemented and this pull request adds a correct implementation of multi-line f-strings to improve logged messages/exceptions

### CODE REVIEW

**Comment by tudorcebere** [comment]
Hello @steph-en-m!

Thanks for the PR, one mention is that we are updating our `black` version for linting. After the update, please rerun linting and commit your changes. Otherwise, LGTM! :+1:

**Comment by steph-en-m** [comment]
Do you have any idea when the updates(black) will be completed?

File: syft/workers/base.py [hunk 1]
```diff
@@ -186,9 +186,6 @@ def get_obj(self, obj_id: Union[str, int]) -> object:
         """
         return self.object_store.get_obj(obj_id)
 
-    def register_obj(self, obj):
```
Comment (gmuraru) [inline syft/workers/base.py:189]: Q: I think this needs to be put back (I think there was another PR merged with this change)

Comment (steph-en-m) [reply syft/workers/base.py]: If I've checked properly, the duplicate method still exists.

**Comment by cereallarceny** [comment]
Hello! Just letting you know that we are no longer planning on supporting anything on the 0.2.x product line and that all work should be ported over to 0.3.x, which is considered a complete rebuild of PySyft. Because of that, I'll be closing this issue. If you feel this is a mistake, or if the issue actually applies to 0.3.x as well, please feel free to ping me on Slack and I'll reopen the issue.

**Pull request closed without merging**